src/pages/Game.js

```diff
@@ -394,20 +394,6 @@ class Game extends Component {
     this.setState({ messageList: [...this.state.messageList, message] });
   }
 
-  getUserInfo = async () => {
```

Comment:
Okay you removed this, but you didn't import the function from utils. This broke the game playing page, our main feature

Reply:
I factored the function into a separate file because it's used in multiple files. I don't think I actually made any changes to game.js